geoengine/raster.py

```diff
@@ -9,6 +9,7 @@
 import geoengine.types as gety
 
```

Comment:
In the example notebook, you always do 

```
axs[2].set_title("Natural Earth blue")
axs[2].set_xlim(-180, 180)
axs[2].set_ylim(-90, 90)
```

on all the rasters. But why don't you call `raster_stream_into_xarray` with `clip_to_query_rectangle=true`?

Reply:
61d32c6528061342807e2c9961eff4b2763ff085

Reply:
![grafik](https://github.com/geo-engine/geoengine-python/assets/4947061/6f458dfd-e326-40ea-84cf-8efb019ad8cd)

Seems broken. It doesn't clip like before but shows lots of NODATA.